Assets/Code/MinigameTanglesLair/GameData.cs

```diff
@@ -89,6 +92,7 @@ public static void setCurrLevel(int l)
     }
 
     public static bool[] getCompletedLevels() {
+        Debug.Log("GameData.cs -> getCompletedLevels:" + saveData.completedLevels);
```

Comment:
Getters (and setters to an extent) are one of the places that you'll want to avoid adding log statements. It can really start to blow up the log with messages as well as reduce performance.